engine/platform/sdl/vid_sdl.c

```diff
@@ -1093,8 +1097,12 @@ rserr_t R_ChangeDisplaySettings( int width, int height, window_mode_t window_mod
 {
 #if SDL_VERSION_ATLEAST( 2, 0, 0 )
 	SDL_DisplayMode displayMode;
+	int displayIndex = SDL_GetWindowDisplayIndex(host.hWnd);
```

Comment:
`host.hWnd` might be NULL here, which seems to be fine for SDL and it will return -1 in this case.

However, is `0` the desired value here?

Similarly to how engine remembers it's window position, why not try to deduce the display index from that? There seems to be `SDL_GetPointDisplayIndex` function for that.